test_scripts/API/VehicleData/GpsShiftSupport/009_GetInteriorVehicleData_stationLocation_shift_after_subscription.lua

```diff
@@ -0,0 +1,84 @@
+--   "moduleData" parameter for the first request
+-- 2) Send a GetInteriorVehicleData response without "shifted" item for the second request
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common = require('test_scripts/API/VehicleData/GpsShiftSupport/commonGpsShift')
+local commonRC = require("test_scripts/RC/commonRC")
+
+-- [[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+-- [[ Local Functions ]]
+local function getInteriorVehicleData(pShiftValue)
+    common.radioData.radioControlData.sisData.stationLocation.shifted = pShiftValue
+    local cid = common.getMobileSession():SendRPC("GetInteriorVehicleData", {
+        moduleType = "RADIO",
+        subscribe = true
+    })
+    EXPECT_HMICALL("RC.GetInteriorVehicleData", {
+        moduleType = "RADIO"
+      })
+    :Do(function(_, data)
+        common.getHMIConnection():SendResponse(data.id, data.method, "SUCCESS", {
+            moduleData = common.radioData,
+            isSubscribed = true
+          })
+    end)
+
+    local expectedRadioData = common.cloneTable(common.radioData)
+    expectedRadioData.radioControlData.sisData.stationLocation.shifted = nil
+    common.getMobileSession():ExpectResponse(cid, { success = true, resultCode = "SUCCESS",
+      isSubscribed = true,
+      moduleData = expectedRadioData
+    })
+    :ValidIf(function(_, data)
+        return common.checkShifted(data.payload.moduleData.radioControlData.sisData.stationLocation.shifted, nil)
+    end)
+end
+
+local function getInteriorVehicleDataSubscribed()
+    local cid = common.getMobileSession():SendRPC("GetInteriorVehicleData", {
+        moduleType = "RADIO"
+    })
+    EXPECT_HMICALL("RC.GetInteriorVehicleData", {
+        moduleType = "RADIO"
+      })
+    :Times(0)
+
+    local expectedRadioData = common.cloneTable(common.radioData)
+    expectedRadioData.radioControlData.sisData.stationLocation.shifted = nil
+    common.getMobileSession():ExpectResponse(cid, { success = true, resultCode = "SUCCESS",
+      moduleData = expectedRadioData
+    })
+    :ValidIf(function(_, data)
+        return common.checkShifted(data.payload.moduleData.radioControlData.sisData.stationLocation.shifted, nil)
+    end)
+end
+
+--[[ Scenario ]]
+runner.Title("Preconditions")
+runner.Step("Clean environment", commonRC.preconditions)
+runner.Step("Start SDL, HMI, connect Mobile, start Session", common.start)
+runner.Step("Register App", common.registerAppWOPTU)
+runner.Step("Activate App", common.activateApp)
+
+runner.Title("Test")
+runner.Step("GetInteriorVehicleData subscription RADIO module, shifted true", getInteriorVehicleData, { true })
+runner.Step("GetInteriorVehicleData RADIO module subscribed, without shifted", getInteriorVehicleDataSubscribed)
+
+runner.Title("Postconditions")
+runner.Step("Stop SDL", common.postconditions)
```

Comment:
Test corrupts the preloaded policy table without the following fix
```suggestion
runner.Step("Stop SDL", commonRC.postconditions)
```